testing/tlp/APKBUILD

```diff
@@ -2,14 +2,15 @@
 # Maintainer: Ivan Tham <pickfire@riseup.net>
 pkgname=tlp
 _pkgname=TLP
-pkgver=0.9
-pkgrel=1
+pkgver=1.1
+pkgrel=0
 pkgdesc="Linux Advanced Power Management"
 url="http://linrunner.de/en/tlp/tlp.html"
 arch="noarch"
-license="GPL-2.0"
+license="GPL-2.0+"
```

Comment:
We are using [SPDX license names](https://spdx.org/licenses/) nowadays. Please change this to `GPL-2.0-or-later`

Reply:
From ML
> After discussing with jirutka, we came to the conclusion that SPDX 2
shorthand identifiers are fine as long as they are not vague. For
example "GPL-2.0+" is equally valid to "GPL-2.0-or-later". This
resolves the main gripe that Adelie has with SPDX 3.